.travis.yml

```diff
@@ -11,37 +16,28 @@ env:
   global:
     - PATH=/opt/python/3.6.7/bin:$PATH
```

Comment:
Related to previous comment. Can we set job-specific env? This only seems applicable to the ansible e2e job.